framework_tvb/tvb/core/services/operation_service.py

```diff
@@ -261,7 +262,7 @@ def initiate_prelaunch(self, operation, adapter_instance):
             msg = "Could not launch Operation with the given input data!"
             self._handle_exception(excep1, temp_files, msg, operation)
 
-        if operation.fk_operation_group and 'SimulatorAdapter' in operation.algorithm.classname:
+        if operation.fk_operation_group and 'SimulatorAdapter' in operation.algorithm.classname and nr_datatypes == 1:
```

Comment:
I remember there was a better reason when I working on this.
The code inside the if should be executed only at PSE launching, not at groups importing as well. If the nr_datatypes == 1 condition is not there, operations are going to be tried to be launched at group importing but the first one will generated an IndexError because there is no BurstConfig (line 157) so the code will be exited and caught later and in the end, the result is the same. Probably some ulterior changes made this happen...

Should I keep it or not?